modules/400-descheduler/crds/deschedulers.yaml

```diff
@@ -0,0 +1,584 @@
+                                      type: array
+                                  type: object
+                                nodeFit:
+                                  type: boolean
+                                removeDuplicates:
+                                  properties:
+                                    excludeOwnerKinds:
+                                      items:
+                                        type: string
+                                      type: array
+                                  type: object
+                                thresholdPriority:
+                                  format: int32
+                                  type: integer
+                                thresholdPriorityClassName:
+                                  type: string
+                              type: object
+                          type: object
+                        removeFailedPods:
+                          properties:
+                            params:
+                              properties:
+                                labelSelector:
+                                  properties:
+                                    matchExpressions:
+                                      items:
+                                        properties:
+                                          key:
+                                            type: string
+                                          operator:
+                                            type: string
+                                          values:
+                                            items:
+                                              type: string
+                                            type: array
+                                        required:
+                                          - key
+                                          - operator
+                                        type: object
+                                      type: array
+                                    matchLabels:
+                                      additionalProperties:
+                                        type: string
+                                      type: object
+                                  type: object
+                                  x-kubernetes-map-type: atomic
+                                namespaces:
+                                  properties:
+                                    exclude:
+                                      items:
+                                        type: string
+                                      type: array
+                                    include:
+                                      items:
+                                        type: string
+                                      type: array
+                                  type: object
+                                nodeFit:
+                                  type: boolean
+                                removeFailedPods:
```

Comment:
Why in all policies are there objects with the same name, e.g.,`removeFailedPods: {removeFailedPods: {}}`

Reply:
Not relevant.